index.html

```diff
@@ -0,0 +1,175 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Oscafest Summit</title>
+		<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.5.0/font/bootstrap-icons.css">
+		<link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <header>
+		<div class="header--social">
+			<ul>
+				<li><a href="#"><i class="bi bi-facebook"></i></a></li>
+        <li><a href="#"><i class="bi bi-twitter"></i></a></li>
+        <li><a href="#">English</a></li>
+        <li><a href="#">My Page</a></li>
+        <li><a href="#">Logout</a></li>
+			</ul>
+		</div>
+		<div class="header--nav--section">
+			<div class="header--hamburger">
+				<img src="images/hamburger.png" alt="Hamburger icon">
+			</div>
+			<a href="/index.html" class="header--logo">
+				<div class="logo--design">
+						<p>OS</p>
+				</div>
+				<p class="logo--text">OS Summit <br>2023</p>
+			</a>
+			<div class="header--nav--container">
+				<nav class="header--nav">
+					<img src="images/Icon.png" class="nav--close--icon" alt="Close Icon">
```

Comment:
- [ ] please kindly add margin left and right to your navigation bars so they can match the original design. Also please align the logo with the text and heading of the page. always pay attention to such small details.
yours
![Screenshot (1184)_LI](https://github.com/Akorede3133/oscafest-conference/assets/65298048/deb96ac9-dca2-43be-a170-acb4cc9c5537)
original
![Screenshot (1026)](https://github.com/Akorede3133/oscafest-conference/assets/65298048/e1e2d12f-eb32-4107-a7f6-b87bc03af659)